### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by itaileshem**

### CODE REVIEW

File: beta/src/sidebarHome.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
           "title": "Community",
           "path": "/community",
           "routes": [{
-            "title": "Acknowledgements",
+            "title": "Acknowledgments",
```
Comment (gaearon) [inline beta/src/sidebarHome.json:18]: If you change this, please also change the link, the title on the page, etc.

Comment (itaileshem) [reply beta/src/sidebarHome.json:18]: Right, changed it also

Comment (itaileshem) [reply beta/src/sidebarHome.json:18]: Right!
Changed everything under beta.

**Comment by harish-sethuraman** [comment]
Can you please rebase?

**Review by import-brain — APPROVED** [review]

**Pull request closed without merging**